ee_site/apps/main/forms.py

```diff
@@ -124,18 +123,13 @@ def process_offer(self):
         ).send()
 
         lead = {
+            'email': self.cleaned_data['reply_to'],
             'company': self.cleaned_data['organization_name'],
             'status': 'new',
             'lastname': '-',
+            'desire__c': self.cleaned_data['message'],
```

Comment:
czemu  message to `desire__c`?

Reply:
to takie customowe pole w naszym salesforce oznaczające czego klient (lead) pragnie. nie było żadnego generycznego odpowiednika

Reply:
no to git